Error: No tags found

I keep getting `Error: No tags found` when running `conventional-github-releaser -p angular`

I can echo my env var `$CONVENTIONAL_GITHUB_RELEASER_TOKEN`.

I created a test repo to test things out:

``` shell
± |master ✓| → git ls-remote
From git@github.com:psyrendust/release-test.git
6a34dcd83951213a96f51736694cbcfa7054fa2f    HEAD
fab70e59230304040bf9375971a6cfb44ff004f7    refs/heads/develop
6a34dcd83951213a96f51736694cbcfa7054fa2f    refs/heads/master
b0e68f75230083b337a9506b43376070abf047d0    refs/tags/v0.0.1
fa59d84ed08fae601b33fc3f3c259c8a50489480    refs/tags/v0.0.1^{}
4ae42acf35833f63e63e70925b8af9dd2f17e687    refs/tags/v0.0.2
6a34dcd83951213a96f51736694cbcfa7054fa2f    refs/tags/v0.0.2^{}
```

I have a few tags, a develop and master branch. I've checked the access permissions of my personal access token and everything is fine there.

I just globally installed version 1.1.1 of `git-semver-tags` and ran the command in my test repo and it doesn't output any tags.

I'm using the following shell script to automate my release process:

``` shell
#!/usr/bin/env bash
# define branches
develop="develop"
master="master"

gitCurrBranch() {
  ref=$(command git symbolic-ref HEAD 2> /dev/null) || \
  ref=$(command git rev-parse --short HEAD 2> /dev/null) || return
  echo "${ref#refs/heads/}"
}

publish() {
  # start with develop branch and make sure that master and develop
  # have both been rebased against each other
  currBranch=`gitCurrBranch` &&
  [ "$currBranch" != $master ] && git checkout $master;
  git rebase $develop &&

  # run tests
  # travis status --no-interactive &&
  trash node_modules &>/dev/null;
  npm install &&
  npm run test &&

  # bump version and build changelog
  cp package.json _package.json &&
  preset=`conventional-commits-detector` &&
  echo $preset &&
  bump=`conventional-recommended-bump -p angular` &&
  echo ${1:-$bump} &&
  npm --no-git-tag-version version ${1:-$bump} &>/dev/null &&
  conventional-changelog -i CHANGELOG.md -w -p ${2:-$preset} &&
  git add CHANGELOG.md &&
  version=`cat package.json | json version` &&

  # commit changes
  git commit -m"docs(CHANGELOG): $version" &&
  mv -f _package.json package.json &&
  npm version ${1:-$bump} -m "chore(release): %s" &&

  # push changes to remote
  git push origin $master &&
  git push origin $develop &&
  git push --tags

  # Update github releases
  conventional-github-releaser -p ${2:-$preset}

  # rebase master onto develop
  git checkout $develop &&
  git rebase $master &&
  git push origin $develop
}
```

running the tests on my machine fails. I'll fix and submit a pull request on that repo.

```shell
± |master ✓| → npm run test

> git-semver-tags@1.1.1 test /Users/larrygordon/dev/westfield/libs/git-semver-tags
> npm run-script lint && mocha --timeout 10000


> git-semver-tags@1.1.1 lint /Users/larrygordon/dev/westfield/libs/git-semver-tags
> jshint *.js --exclude node_modules && jscs *.js

No code style errors found.


  ✓ should error if no commits found
  ✓ should get no semver tags (224ms)
  1) should get the semver tag
  2) should get both semver tags
  3) should get all semver tags if two tags on the same commit
  4) should still work if I run it again
  5) should be in reverse chronological order
  6) should work with prerelease
  7) should work with empty commit

  2 passing (2s)
  7 failing

  1)  should get the semver tag:

      Uncaught AssertionError: [] deepEqual [ 'v2.0.0' ]
      + expected - actual

      -[]
      +[
      +  "v2.0.0"
      +]

      at test.js:44:5
      at index.js:28:5
      at ChildProcess.exithandler (child_process.js:194:7)
      at maybeClose (internal/child_process.js:818:16)
      at Process.ChildProcess._handle.onexit (internal/child_process.js:211:5)

  2)  should get both semver tags:

      Uncaught AssertionError: [] deepEqual [ 'v3.0.0', 'v2.0.0' ]
      + expected - actual

      -[]
      +[
      +  "v3.0.0"
      +  "v2.0.0"
      +]

      at test.js:54:5
      at index.js:28:5
      at ChildProcess.exithandler (child_process.js:194:7)
      at maybeClose (internal/child_process.js:818:16)
      at Process.ChildProcess._handle.onexit (internal/child_process.js:211:5)

  3)  should get all semver tags if two tags on the same commit:

      Uncaught AssertionError: [] deepEqual [ 'v4.0.0', 'v3.0.0', 'v2.0.0' ]
      + expected - actual

      -[]
      +[
      +  "v4.0.0"
      +  "v3.0.0"
      +  "v2.0.0"
      +]

      at test.js:64:5
      at index.js:28:5
      at ChildProcess.exithandler (child_process.js:194:7)
      at maybeClose (internal/child_process.js:818:16)
      at Process.ChildProcess._handle.onexit (internal/child_process.js:211:5)

  4)  should still work if I run it again:

      Uncaught AssertionError: [] deepEqual [ 'v4.0.0', 'v3.0.0', 'v2.0.0' ]
      + expected - actual

      -[]
      +[
      +  "v4.0.0"
      +  "v3.0.0"
      +  "v2.0.0"
      +]

      at test.js:72:5
      at index.js:28:5
      at ChildProcess.exithandler (child_process.js:194:7)
      at maybeClose (internal/child_process.js:818:16)
      at Process.ChildProcess._handle.onexit (internal/child_process.js:211:5)

  5)  should be in reverse chronological order:

      Uncaught AssertionError: [] deepEqual [ 'v1.0.0', 'v4.0.0', 'v3.0.0', 'v2.0.0' ]
      + expected - actual

      -[]
      +[
      +  "v1.0.0"
      +  "v4.0.0"
      +  "v3.0.0"
      +  "v2.0.0"
      +]

      at test.js:84:5
      at index.js:28:5
      at ChildProcess.exithandler (child_process.js:194:7)
      at maybeClose (internal/child_process.js:818:16)
      at Process.ChildProcess._handle.onexit (internal/child_process.js:211:5)

  6)  should work with prerelease:

      Uncaught AssertionError: [] deepEqual [ '5.0.0-pre', 'v1.0.0', 'v4.0.0', 'v3.0.0', 'v2.0.0' ]
      + expected - actual

      -[]
      +[
      +  "5.0.0-pre"
      +  "v1.0.0"
      +  "v4.0.0"
      +  "v3.0.0"
      +  "v2.0.0"
      +]

      at test.js:96:5
      at index.js:28:5
      at ChildProcess.exithandler (child_process.js:194:7)
      at maybeClose (internal/child_process.js:818:16)
      at Process.ChildProcess._handle.onexit (internal/child_process.js:211:5)

  7)  should work with empty commit:

      Uncaught AssertionError: [] deepEqual [ 'v1.1.0' ]
      + expected - actual

      -[]
      +[
      +  "v1.1.0"
      +]

      at test.js:111:5
      at index.js:28:5
      at ChildProcess.exithandler (child_process.js:194:7)
      at maybeClose (internal/child_process.js:818:16)
      at Process.ChildProcess._handle.onexit (internal/child_process.js:211:5)




npm ERR! Darwin 15.3.0
npm ERR! argv "/Users/larrygordon/.nvm/versions/node/v4.2.2/bin/node" "/Users/larrygordon/.nvm/versions/node/v4.2.2/bin/npm" "run" "test"
npm ERR! node v4.2.2
npm ERR! npm  v2.14.14
npm ERR! code ELIFECYCLE
npm ERR! git-semver-tags@1.1.1 test: `npm run-script lint && mocha --timeout 10000`
npm ERR! Exit status 7
npm ERR!
npm ERR! Failed at the git-semver-tags@1.1.1 test script 'npm run-script lint && mocha --timeout 10000'.
npm ERR! This is most likely a problem with the git-semver-tags package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     npm run-script lint && mocha --timeout 10000
npm ERR! You can get their info via:
npm ERR!     npm owner ls git-semver-tags
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/larrygordon/dev/westfield/libs/git-semver-tags/npm-debug.log
```

--- a/index.js
+++ b/index.js
@@ -2,7 +2,7 @@
 var exec = require('child_process').exec;
 var semverValid = require('semver').valid;
 var regex = /tag:\s*(.+?)[,\)]/gi;
-var cmd = 'git log --decorate';
+var cmd = 'git log --decorate --no-color';
 
 module.exports = function(callback) {
   exec(cmd, {

